tests/engine/test_arg_utils.py

```
@@ -360,3 +360,22 @@ def test_human_readable_model_len():
     for invalid in ["1a", "pwd", "10.24", "1.23M"]:
         with pytest.raises(ArgumentError):
             args = parser.parse_args(["--max-model-len", invalid])
+
+
+def test_hf_overrides():
+    args = [
+        "--hf_overrides", '{"matryoshka_dimensions":[256]}',
+        "--hf_overrides.matryoshka_dimensions", "[256]"
+    ]
+    parser = EngineArgs.add_cli_args(
+        FlexibleArgumentParser(exit_on_error=False))
```

Why do we ignore errors here?